components/cards/cards.css

```diff
@@ -61,6 +61,82 @@
 	right: 2%;
 	top: 2%;
 }
+
+.product-card {
+	width: 24rem;
+	margin-top: 5rem;
+	border-radius: 0.5rem;
+}
+
+.product-card > a > img {
```

Comment:
I suggest don't use selectors, instead use the direct class names for a particular element. 
ex. : .product-card ( which is first)
        .product-card-link ( you use '> a' )
        .product-card-img (you use '> img')